src/renderer/containers/Onboard/DepositTx/DepositTxContainer.tsx

```diff
@@ -17,77 +20,139 @@ const depositContracts = [
     public render(): ReactElement {
-        const networkOptions = depositContracts.map((contract) => {return contract.networkName;});
+        const networkOptions = depositContracts.map((contract) => { return contract.networkName; });
         const selectedContract = depositContracts[this.state.selectedNetworkIdx];
+        const {txData} = this.props.deposit;
+
         return (
             <>
                 <h1>Deposit transaction</h1>
                 <p>Execute this transaction from your ETH1 wallet of choice along with 32 ETH to become validator.</p>
                 <div className="deposit-details-container">
                     <Dropdown
-                        label="Network" 
-                        current={this.state.selectedNetworkIdx} 
-                        onChange={this.onNetworkChange} 
+                        label="Network"
+                        current={this.state.selectedNetworkIdx}
+                        onChange={this.onNetworkChange}
                         options={networkOptions} />
                     <CopyField
                         label="Deposit contract"
                         value={selectedContract.address}
                         onCopy={(): void => copyToClipboard(selectedContract.address)} />
                     <CopyField
                         label="Transaction data"
-                        value={this.state.transactionData}
-                        onCopy={(): void => copyToClipboard(this.state.transactionData)} />
+                        value={txData}
+                        onCopy={(): void => copyToClipboard(txData)} />
                 </div>
                 <div className="deposit-action-buttons">
-                    <ButtonSecondary buttonId="skip">SKIP</ButtonSecondary>
-                    <ButtonPrimary buttonId="verify">Verify</ButtonPrimary>
+                    <ButtonSecondary
+                        buttonId="skip"
+                        onClick={this.handleSkip}
+                    >
+                        SKIP
+                    </ButtonSecondary>
+                    <ButtonPrimary
+                        buttonId="verify"
+                        onClick={this.handleVerify}
+                    >
+                        Verify
+                    </ButtonPrimary>
                 </div>
             </>
         );
     }
+
+    private handleSkip = (): void => {
+        this.props.history.push(Routes.ONBOARD_ROUTE_EVALUATE(OnBoardingRoutes.PASSWORD));
+    };
+
+    // TODO there is a flag in redux "isDepositVisible" so component should wait until flag is set to true
+    private handleVerify = (): void => {
+        const {selectedContractAddress, selectedNetworkIdx} = this.state;
+        // FIXME  pass network name from select but find names that ether.js supports
+        // You can use any standard network name
+        //  - "homestead"
+        //  - "rinkeby"
+        //  - "ropsten"
+        //  - "kovan"
+        //  - "goerli"
```

Comment:
We need eth1NetworkId in networkConfig and we need to have some utility function that maps that it to ethers network name